docs/tutorials/projects/helmet.md

```diff
@@ -0,0 +1,455 @@
+Configure data capture on the `objectfilter` camera to capture images of people without hard hats:
+
+1. First, you need to add the data service to your machine to make it available to capture data on your camera.
+
+   Navigate to the **Services** subtab of your machine's **Config** tab.
+
+   Click **Create service**.
+   Type "data" and click **data management / RDK**.
+   Name your data management service `data-manager` and click **Create**.
+
+   Leave all the default data service attributes as they are and click **Save config**.
+
+2. Now you're ready to enable data capture on your detector camera.
+   Navigate to the **Components** subtab.
+   Locate the `objectfilter-cam` panel.
+
+3. Click **Add method**.
+   Click the **Type** dropdown and select **ReadImage**.
+   Set the capture frequency to `0.2` images per second (equivalent to one image every 5 seconds).
+   You can always change the frequency to suit your use case.
+
+   {{<imgproc src="/tutorials/helmet/datacapture-config.png" resize="x1100" declaredimensions=true alt="The objectfilter cam configured with data capture set to read images at 0.2 hertz. Mime type is left at the default image/jpeg." >}}
+
+### Test data capture and sync
+
+To make sure the detector camera is capturing and syncing labeled images:
+
+1. Position yourself in front of your webcam for 30 seconds or so to let it capture a few images of a person without a hard hat on.
+
+2. Navigate to your [**DATA** page](https://app.viam.com/data/view?view=images) in the Viam app.
+   You should see some images with bounding boxes on them.
+   If you do not, try refreshing the page.
+
+   {{<imgproc src="/tutorials/helmet/synced-data.png" resize="x1000" declaredimensions=true alt="The data manager page in the Viam app, displaying three images of a person with a bounding box labeled NO-Hardhat around her face." >}}
+
+3. You can also try this with a hard hat on your head.
+
+   {{<imgproc src="/tutorials/helmet/hardhat-data.png" resize="x1000" declaredimensions=true alt="The data manager page in the Viam app, displaying three images of a person with a bounding box labeled NO-Hardhat around her face." >}}
+
+### Modify the detector config to sync only images without hard hats
+
+Until now, you've been identifying people without hard hats as well as people with hard hats.
+Now that you have verified that the detector and data sync are working, modify your config so that only images with people _without_ hard hats are captured:
+
+1. Navigate to your `objectfilter-cam` card on the **Config** tab.
+
+2. Delete the `"Hardhat",` line from the `"labels"` array.
+
+3. Click **Save config**.
+
+## Set up email notifications
+
+[Webhooks](https://en.wikipedia.org/wiki/Webhook) are a way to trigger a custom function to run when a certain event happens.
+For the purposes of this project, you will use a webhook to run a function that sends you an email when an image of someone without a hard hat is uploaded to the cloud.
```

Comment:
Let's link to our own docs. Webhooks I think are a request, it's not necessarily a function that gets executed. I've 
```suggestion
[Webhooks](/build/configure/#webhooks) allow you to trigger actions by sending an HTML request when a certain event happens.
In this case, you're going to set up a webhook to trigger a serverless function that sends you an email when an image of someone without a hard hat is uploaded to the cloud.
```